packages/components/ng-ovh-stop-event/src/directive.js

```diff
@@ -1,9 +1,12 @@
-export default function () {
+export default function() {
```

Comment:
Did we forget to run prettier on those files ?

Reply:
I had some problem running husky hook on commit. Seems to be fixed now. I will run it to correct it.